src/actions/auth.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios"
+import actionTypes from "./actionTypes";
+const BASE_URL = "https://misocho01-questioner.herokuapp.com/api/v2";
+
+export const login = (username, password, props) => {
+    return axios
+        .post(`${BASE_URL}/auth/signin`, {
+            username,
+            password
+        })
```

Comment:
You can decouple this into a helper function and have it in a helper file so that you can reuse it across all the actions. The function will take parameters such as `method`, `payload` and `endpoint`. Let me know if you need me to give more insight on the same.